lib/fragment.js

```diff
@@ -47,7 +48,7 @@ module.exports = class Fragment extends EventEmitter {
     constructor (tag, context, index, requestFragment, pipeInstanceName) {
         super();
         this.attributes = getAttributes(tag, context);
-        ['async', 'primary', 'public'].forEach((key) => {
+        ['async', 'primary', 'public', 'main'].forEach((key) => {
```

Comment:
we brainstormed up a better idea - instead of baking in the concept of 'main' timing, to have generic timing groups that are programmatically available to the developer.